packages/roosterjs-editor-dom/lib/htmlSanitizer/HtmlSanitizer.ts

```diff
@@ -10,6 +10,7 @@ import toArray from '../jsUtils/toArray';
 import { cloneObject } from './cloneObject';
 import { isCssVariable, processCssVariable } from './processCssVariable';
 import { NodeType } from 'roosterjs-editor-types';
+import { splitSelectors } from 'roosterjs-content-model-core';
```

Comment:
Do not import from content model code, but make a duplication here.

Old code and new code need to be totally separated. Old code will be deprecated soon.